cpp/Contact.hpp

```diff
@@ -83,37 +84,35 @@ class Contact
       return cell_facet_pairs;
     };
 
-    _cell_facet_pairs_0 = get_cell_indices(_facet_0);
-    _cell_facet_pairs_1 = get_cell_indices(_facet_1);
+    // Replace with loop once it is generalized to an arbitrary number of
+    // surfaces
+    _cell_facet_pairs[0] = get_cell_indices(_facets[0]);
+    _cell_facet_pairs[1] = get_cell_indices(_facets[1]);
   }
 
-  const std::vector<int32_t>& facet_0() const { return _facet_0; }
-  const std::vector<int32_t>& facet_1() const { return _facet_1; }
-  // Return meshtag value for surface_0
-  const int surface_0() const { return _surface_0; }
-  // Return mestag value for suface_1
-  const int surface_1() const { return _surface_1; }
+  const std::vector<int32_t>& facets(int surface) const
+  {
+    return _facets[surface];
+  }
+  // Return meshtag value for surface
+  const int surface_mt(int surface) const { return _surfaces[surface]; }
   // return quadrature degree
   const int quadrature_degree() const { return _quadrature_degree; }
   void set_quadrature_degree(int deg) { _quadrature_degree = deg; }
 
   // return distance map
   const std::shared_ptr<dolfinx::graph::AdjacencyList<std::int32_t>>
```

Comment:
Maybe improve this doc-string a bit as we now only have a single function for both maps. Would be good to know what each maps from and to.